APIURLS.md

```diff
@@ -0,0 +1,110 @@
+Admin Features:
+    Manage Project:
+        Projects Database:
+            localhost:8080/API/v1/project/id/1
+            [Post, Put, Get, Delete]
+        Create Project:
+            POST localhost:8080/API/v1/projects/id
+        Search Project:
+            GET localhost:8080/API/v1/projects?search=id
+        Delete Project:
+            Delete localhost:8080/API/v1/projects/id
+        List Project Table:
+            GET localhost:8080/API/v1/projects
+    User Management:
+        Users Database:
+            localhost:8080/API/v1/User/id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/Users
+        Invite Member: (send user invitation by email)
+            POST localhost:8080/API/v1/projects/id/Users/id
+        List User Table:
+            GET localhost:8080/API/v1/projects/id/Users
+        User status:
+            GET localhost:8080/API/v1/Users/id/status
+        Update User Last active:
+            GET localhost:8080/API/v1/Users/id/lastActive
+            PATCH localhost:8080/API/v1/Users/id
+        User Detailed Name Card:
+            GET localhost:8080/API/v1/Users/id/fullname
+            GET localhost:8080/API/v1/Users/id/emailaddress
+            GET localhost:8080/API/v1/Users/id/publicname
+    Task Board Management:

```

Comment:
Missing API `Status update` -- basically moved from one column to another

Reply:
This one I still don't see an PATCH API for TaskCard move from one column to another in the Main kanban board